<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSessionUI</class>
 <widget class="QWidget" name="GameSessionUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>276</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tilitoli - Játéktábla</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(122, 126, 136);</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(122, 119, 155);
background-color: rgb(122, 119, 125);
color: rgb(41, 13, 56);</string>
   </property>
   <property name="text">
    <string>Főmenü</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>111</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="stepCountLabel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Lépések száma:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="stepCounter">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="TableView" name="gameTable" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>161</width>
     <height>149</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(52, 66, 76);</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QWidget</extends>
   <header location="global">Headers/tableview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
